Description: Fix tracker-extract.service WantedBy target
 Having WantedBy=default.target was triggering the service before the session
 environment variable are set in gnome-session.
 .
 This service relies on gvfs-daemon.service and by starting it before the
 environment variables are set, some important variables like KRB5CCNAME are
 missing. This means that the daemon won't have access to it when trying to
 mount shared folders through the network, creating problems like requesting 
 credentials when trying to gio mount a samba share, despite having a valid
 kerberos ticket on the machine.
Forwarded: 
Bug: https://gitlab.gnome.org/GNOME/gvfs/-/issues/481
Bug-Ubuntu: https://launchpad.net/bugs/1779890
Author: Denison Barbosa <denison.barbosa@canonical.com>
Last-Update: 2023-03-21

Index: tracker-miners-3.4.3/src/tracker-extract/tracker-extract.service.in
===================================================================
--- tracker-miners-3.4.3.orig/src/tracker-extract/tracker-extract.service.in
+++ tracker-miners-3.4.3/src/tracker-extract/tracker-extract.service.in
@@ -1,6 +1,7 @@
 [Unit]
 Description=Tracker metadata extractor
 ConditionUser=!root
+After=gnome-session.target
 
 [Service]
 Type=dbus
@@ -12,4 +13,4 @@ RestartPreventExitStatus=SIGKILL
 Slice=background.slice
 
 [Install]
-WantedBy=default.target
+WantedBy=gnome-session.target
